manifests/backend.pp

```diff
@@ -64,8 +71,14 @@
   Optional[String] $agent_old_password = undef,
   Boolean $include_default_resources = true,
   Boolean $show_diff = true,
+  Optional[String] $license_source = undef,
+  Optional[String] $license_content = undef,
 ) {
 
+  if $license_source and $license_content {
+    fail('sensu::backend: Do not define both license_source and license_content')
```

Comment:
I believe the error will already report that it came from the sensu::backend class.

Reply:
Maybe if run from master but with `puppet apply` it does not when I removed reference to `sensu::backend: `.

```
Error: Evaluation Error: Error while evaluating a Function Call, Do not define both license_source and license_content (file: /etc/puppetlabs/code/environments/production/modules/sensu/manifests/backend.pp, line: 79, column: 5) on node sensu-backend.example.com
```